mod.sp

```diff
@@ -7,6 +7,76 @@ locals {
   }
 }
 
+variable "common_dimensions" {
+  type        = list(string)
+  description = "A list of common dimensions to add to each control."
+  # Define which common dimensions should be added to each control.
+  # - connection_name (_ctx ->> 'connection_name')
+  # - region
+  # - resource_group
+  # - subscription
+  # - subscription_id
+  default = ["resource_group", "subscription"]
```

Comment:
@Priyanka-Chatterjee-2000 Should we keep `tenant_id` in default as it is not giving tenant_id for IAM controls that are using azuread_* tables?